solidity-v1/dashboard/src/css/upgrade-tokens-tile.less

```diff
@@ -0,0 +1,34 @@
+.upgrade-tokens-tile {
```

Comment:
If you agree with comment about `UpgradeTokensTile` we can get rid off this file.

Reply:
Please see https://github.com/keep-network/keep-core/pull/2668#discussion_r753091496